buildSrc/src/test/kotlin/com/datadog/gradle/plugin/jsonschema/ModelValidationTest.kt

```diff
@@ -62,17 +62,26 @@ class ModelValidationTest(
     fun `validate model serialization and deserialization`() {
         val type = Class.forName("com.example.model.$className")
         val toJson = type.getMethod("toJson")
-        val fromJson = type.getMethod("fromJson", String::class.java)
+        val isConstantClass = (className == "Location")
```

Comment:
This hardcoded name in a test is usually a bad practice. You can actually add an additional param to the test class to know if the tested instance is constant or not.